packages/recaf.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,10 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'recaf-2.21.14-J8-jar-with-dependencies.jar'
```

Comment:
The tool name should not include the `.jar`, as this makes this look different than other tools:
![image](https://github.com/user-attachments/assets/ec92a69d-4c0e-41a0-92fc-5ab54efb0206)

The issue template mentions the name should not include the `.exe`, not sure if we should rephrase it to make it clear for other cases (like the one we have here with `.jar`:
>The name of the tool being installed (usually the file name with the .exe), normally different from the package name. Example: FakeNet-NG (tool name) vs fakenet-ng (package name).

Please remove the `.jar`

Reply:
its fixed